modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -49,6 +51,31 @@
   [& args]
   (apply (get-method driver/describe-table :sql-jdbc) args))
 
+(defmethod sql-jdbc.sync/describe-fks-sql :redshift
```

Comment:
We need a test to check if this query filters table name and schema name correctly. Do we need to escape the names?

Reply:
Name escaping is handled by this being a honeysql query.